tools/repo/notify.py

```diff
@@ -185,17 +185,23 @@ async def tracked_prs(self):
             if age > self.slo_max + datetime.timedelta(hours=36):
                 stalled_prs.append(message)
 
+            if self.is_contrib(pull):
+                # If the PR is a contrib PR, we don't want to assign it to maintainer.
```

Comment:
why not? maintainers still need to land these and are usually the right people

Reply:
I don't know, that's already the behavior, I just moved the condition up to here so it doesn't do a bunch of work for no reason in this case.